docs/admin/install/helm-install/microsoft-azure.md

```diff
@@ -3,6 +3,218 @@ tags:
   - administration
   - installation
   - helm
+  - AKS
+  - Microsoft
+  - Azure
 ---
 
-This file is a placeholder.
+# Install Janssen on AKS
+
+## System Requirements
+
+The resources may be set to the minimum as below:
+
+- 8 GiB RAM
+- 8 CPU cores
+- 50GB hard-disk
+
+Use the listing below for detailed estimation of minimum required resources. Table contains the default resources recommendations per service. Depending on the use of each service the resources needs may be increase or decrease.
+
+| Service           | CPU Unit | RAM   | Disk Space | Processor Type | Required                           |
+| ----------------- | -------- | ----- | ---------- | -------------- | ---------------------------------- |
+| Auth server       | 2.5      | 2.5GB | N/A        | 64 Bit         | Yes                                |
+| LDAP (OpenDJ)     | 1.5      | 2GB   | 10GB       | 64 Bit         | Only if couchbase is not installed |
+| fido2             | 0.5      | 0.5GB | N/A        | 64 Bit         | No                                 |
+| scim              | 1.0      | 1.0GB | N/A        | 64 Bit         | No                                 |
+| config - job      | 0.5      | 0.5GB | N/A        | 64 Bit         | Yes on fresh installs              |
+| persistence - job | 0.5      | 0.5GB | N/A        | 64 Bit         | Yes on fresh installs              |
+| client-api        | 1        | 0.4GB | N/A        | 64 Bit         | No                                 |
+| nginx             | 1        | 1GB   | N/A        | 64 Bit         | Yes if not ALB                     |
+| auth-key-rotation | 0.3      | 0.3GB | N/A        | 64 Bit         | No [Strongly recommended]          |
+| config-api        | 0.5      | 0.5GB | N/A        | 64 Bit         | No                                 |
+
+Releases of images are in style 1.0.0-beta.0, 1.0.0-0
+
+## Initial Setup
+
+1.  Install [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+2.  Create a [Resource Group](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli#create-a-resource-group)
+    ```
+    az group create --name myResourceGroup --location eastus
+    ```
+  
+3.  Create an [AKS cluster](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli#create-aks-cluster) such as the following example:
+    ```
+    az aks create -g myResourceGroup -n myAKSCluster --enable-managed-identity --node-vm-size NODE_TYPE --node-count 2 --enable-addons monitoring --enable-msi-auth-for-monitoring  --generate-ssh-keys 
```

Comment:
lets use the name `janssen` across our examples. You can call things like `example-janssen` for our clusters and groups